code/modules/clothing/suits/marine_armor.dm

```diff
@@ -200,6 +199,26 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 	icon_state = "6"
 	armor = list(melee = 30, bullet = 70, laser = 45, energy = 30, bomb = 60, bio = 10, rad = 10)
 
+/obj/item/clothing/suit/storage/marine/M3P/tanker
+	name = "\improper M3 pattern tanker armor"
+	desc = "A modified and refashioned suit of M3 Pattern armor designed to be worn by the loader of a TGMC vehicle crew. While the suit is a bit more encumbering to wear with the crewman uniform, it offers the loader a degree of protection that would otherwise not be enjoyed."
+	icon_state = "tanker"
+
+	New()
```

Comment:
Relative typepath, turn it into
`/obj/item/clothing/suit/storage/marine/M3P/tanker/New()`
and fix indentation